en/ui-system/components/editor/canvas.md

```diff
@@ -1,14 +1,14 @@
+![](canvas/canvas.png)
 
-The Canvas __design resolution__ and __adaptation scheme__ are uniformly set in the __Project Setting__. A camera is provided inside the __Canvas__, and the default z axis direction is __-1000 ~ 998__, so the z axis on the UI must be within this range to display properly (without taking the threshold value).
+The node where the **Canvas** component is the root of the UI rendering. There can be multiple __Canvas__ in the scene. **all UI elements must be placed under a __Canvas__ so that it can be rendered**. Under which Canvas node, it is rendered by which Canvas. Each canvas can be presented in a certain order controlled by the `priority` attribute.
 
-![](canvas/canvas.png)
+In addition to the root node of the UI, it also has a screen adaptation function,  please refer to the [Multi-Resolution Adaption](../engine/multi-resolution.md) documentation. It also plays very important feature in the game production is the multi-resolution adaptation. The Canvas __design resolution__ and __adaptation scheme__ are uniformly set in the __Project Setting__. A camera is provided inside the __Canvas__, and the default z axis direction is __-1000 ~ 998__, so the z axis on the UI must be within this range to display properly (without taking the threshold value).
 
-In the previous design, __Canvas__ was last rendered, meaning it could mask the rendering of all 3D content, but this was far from sufficient for project development needs, such as a 2D map with a 3D character. So, we've added the **RenderMode** property, which allows developers to sort the rendering order of the 3D Camera and UI Camera. To be noted, if you want to have canvas and 3d camera content mixed up, only the lowest camera or canvas can have the **SOLID_COLOR** **ClearFlag**, otherwise a camera with **SOLID_COLOR** flag will erase all content rendered before it.
+![](canvas/design-resolution.png)
 
-The __Canvas__ component is not only the root of the UI rendering, but also a very important feature in the game production is the multi-resolution adaptation, please refer to the [Multi-Resolution Adaption](../engine/multi-resolution.md) documentation.
+In the previous design, __Canvas__ was last rendered, meaning it could mask the rendering of all 3D content, but this was far from sufficient for project development needs, such as a 2D map with a 3D character. So, we've added the **RenderMode** property, which allows developers to sort the rendering order of the 3D Camera and UI Camera. To be noted, if you want to have canvas and 3d camera content mixed up, only the lowest camera or canvas can have the **SOLID_COLOR** **ClearFlag**, otherwise a camera with **SOLID_COLOR** flag will erase all content rendered before it.
```

Comment:
In the previous design, __Canvas__ was last rendered, meaning it could mask the rendering of all 3D content, but this was far from sufficient for project development needs, such as a 2D map with a 3D character. So, we've added the **RenderMode** property, which allows developers to sort the rendering order of the 3D Camera and UI Camera. To be noted, if you want to have canvas and 3d camera content mixed up, only the lowest camera or canvas can have the **SOLID_COLOR** **ClearFlag**, otherwise a camera with **SOLID_COLOR** flag will erase all content rendered before it.
-->
The **RenderMode** property allows developers to sort the rendering order of the 3D Camera and the UI Camera. If you want to have canvas and 3D camera content mixed up, only the lowest camera or canvas can have the **SOLID_COLOR** **ClearFlag**, otherwise a camera with **SOLID_COLOR** flag will erase all content rendered before it.